docs/source/2-install.md

```diff
@@ -219,9 +219,9 @@ conn = flowclient.Connection("https://localhost:9090", "JWT_STRING", ssl_certifi
 cd secrets_quickstart
 docker login
 docker swarm init
-openssl rand -base64 16 | docker secret create FLOWMACHINE_DB_PASS -
+openssl rand -base64 16 | docker secret create FLOWMACHINE_FLOWDB_PASS -
 echo "fm" | docker secret create FLOWMACHINE_DB_USER -
-echo "api" | docker secret create FLOWAPI_DB_USER -
+echo "api" | docker secret create FLOWAPI_FLOWDB_USER -
```

Comment:
Use "analyst" and "reporter" instead of "fm" and "api" here, for consistency with what we use elsewhere?

Reply:
Would be inclined to change elsewhere to reflect this rather than other way round.